### PR TITLE
 Update files

### PR DESCRIPTION
**Pull request by jasjs1**

### CODE REVIEW

File: src/DeveloperAccount/signup/createAccount.js
Comment (shoshta73) [inline src/DeveloperAccount/signup/createAccount.js:1]: ???

**Review by shoshta73 — CHANGES_REQUESTED** [review]
will need to port all this to react

File: src/DeveloperAccount/signup/createAccount.js
Comment (jasjs1) [reply src/DeveloperAccount/signup/createAccount.js:1]: This isnt complete, i am not sure why this got into the pr. I will remove it and fit it at a later time

**Comment by jasjs1** [comment]
> will need to port all this to react

Alright, I am not sure how to do that though.

**Pull request closed without merging**